components/Form/Checkbox/index.js

```diff
@@ -16,7 +16,7 @@ const Checkbox = ({ checked, label, name, onClick, ...others }) => {
             {...others}
           />
           <label className="govuk-label govuk-checkboxes__label" htmlFor={name}>
-            {label}
+            <span hidden>{label}</span>
```

Comment:
The checkbox label definitely shouldnt be hidden

Reply:
True! Forgot we actually have checkboxes that need labels!